src/components/Dialog.tsx

```diff
@@ -175,6 +137,14 @@ export default class Dialog extends Component<DialogProps, DialogState> {
         </Dropdown>
         <br />
         <Button onClick={() => this.props.sdk.close('Done!')}>Done</Button>
+        <br />
+        <br />
+        {this.state.selectedSource && (
+          <Gallery
+            selectedSource={this.state.selectedSource}
+            imgix={this.state.imgix}
+          />
+        )}
```

Comment:
Side note: I just discovered that this condition doesn't doesn't work as intended but fixing it leads to a different issue with Gallery's `componentDidUpdate`. So I'll need to look into that as well.